### PR TITLE
Calculator social media buttons

### PR DESCRIPTION
**Pull request by RoyEJohnson**
Also put them on the Give confirmation page

### CODE REVIEW

**Comment by RoyEJohnson** [comment]
[[OSW2.14.01] Implement Calculator on All Confirmation pages](https://trello.com/c/VWidrej2/36-osw2-14-01-implement-calculator-on-all-confirmation-pages)

**Comment by RoyEJohnson** [comment]
[Facebook share button too large on give confirmation page](https://trello.com/c/Nncx8cO9/166-facebook-share-button-too-large-on-give-confirmation-page)

File: src/app/components/share/share.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import {Controller} from 'superb';
+import {description as template} from './share.html';
+
+/* eslint-disable */
+(function(d, s, id) {
+  var js, fjs = d.getElementsByTagName(s)[0];
+  if (d.getElementById(id)) return;
+  js = d.createElement(s); js.id = id;
+  js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8";
```
Comment (philschatz) [inline src/app/components/share/share.js]: Is the only way to share on facebook to have facebook track page views?

Comment (RoyEJohnson) [inline src/app/components/share/share.js]: As far as I can tell, yes.

Comment (edwoodward) [inline src/app/components/share/share.js]: Are d and s good variable names? I'm not sure what they reference.

Comment (RoyEJohnson) [inline src/app/components/share/share.js]: @edwoodward This is a Facebook-provided snippet to load their SDK. The parameters d, s, and id refer to the supplied arguments document, 'script', and 'facebook-jssdk'.

**Pull request merged**